examples/bigcommerce/README.md

```diff
@@ -203,6 +212,12 @@ const slug = ctx.params?.slug
 const product = await getProduct(Number.parseInt(slug.toString(), 10))
 ```
 
+### Why is the "dynamic product route" using [ISR](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) when the root catchall route is not?
+
+Pages are created in the "optional catchall route" based on pages in a Makeswift site. Since Makeswift is aware of what pages are published it can use [on demand revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#using-on-demand-revalidation) with an api route in `/pages/api/makeswift/[...makeswift].ts` to rebuild pages on publish
+
+Unlike the "optional catchall route", the "dynamic product route" creates pages based on Bigcommerce products. These routes are unknown to Makeswift and thus it doesn't revalidate them on demand. Instead, ISR is used to take care of these updates.
```

Comment:
So are we using a low `revalidate` value for the product page instead of on-demand revalidation now?

Reply:
Yes we are. In fact, I never included on demand because I was forced to use 12.1.6 and I didn't want to use an unstable feature. 

I think its nice to not get into the weed explaining how to setup an on demand route in this example. If we decide to change them all in the future I am not opposed though.